### PR TITLE
Change terminal command to set the TERM env

### PR DESCRIPTION
**Pull request by jwforres**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1367337

### CODE REVIEW

**Comment by jwforres** [comment]
@spadgett PTAL

kubectl / oc rsh command defaults to xterm today if it doesnt find a different TERM env in your cli

**Comment by jwforres** [comment]
this makes things like `clear` actually behave the way you expect

**Comment by spadgett** [comment]
```
bower not-cached    https://github.com/kubernetes-ui/container-terminal.git#1.0.0
bower resolve       https://github.com/kubernetes-ui/container-terminal.git#1.0.0
bower ENORESTARGET  No tag found that was able to satisfy 1.0.0
```

**Comment by spadgett** [comment]
term.js is an xterm clone, so TERM=xterm looks right

**Comment by spadgett** [comment]
Should we update debug-terminal.html as well?

**Comment by jwforres** [comment]
ok updated. I also made the change to not listen for resize till the terminal tab is selected

File: app/scripts/controllers/pod.js [hunk 1]
```diff
@@ -128,17 +128,19 @@ angular.module('openshiftConsole')
...
+        if (!characterBoundingBox.height || !characterBoundingBox.width) {
+          Logger.warn("Unable to calculate the bounding box for a character.  Terminal will not be able to resize.");
+        }
+        else {
+          $(window).on('resize.terminalsize', _.debounce(calculateTerminalSize, 100));
+        }        
         $timeout(calculateTerminalSize, 0);
       }
+      else {
+        $(window).off('resize.terminalsize');
```
Comment (spadgett) [inline app/scripts/controllers/pod.js]: Do we still need the `off()` call in destroy? What if I navigate away with the terminal tab selected?

**Comment by spadgett** [comment]
[merge]

**Pull request merged**